### PR TITLE
Case 22433: master: Query shape type outside lock

### PR DESCRIPTION
**Pull request by Atlante45**
https://highfidelity.manuscript.com/f/cases/22485/v83-Entity-Server-deadlock-attempting-to-restore-from-archive-failing

### CODE REVIEW

File: libraries/networking/src/udt/PacketHeaders.h [hunk 1]
```diff
@@ -269,6 +269,7 @@ enum class EntityVersion : PacketVersion {
     CertificateTypeProperty,
     DisableWebMedia,
     ParticleShapeType,
+    ParticleShapeTypeDeadlockFix,
```
Comment (Atlante45) [inline libraries/networking/src/udt/PacketHeaders.h]: @ZappoMan, want me to rename this?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19148/HighFidelity-Beta-PR15495-ecc38a8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19148/HighFidelity-Quest-Beta-PR15495-ecc38a8.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19148/HighFidelity-Beta-PR15495-ecc38a8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19148/HighFidelity-Beta-Interface-PR15495-ecc38a8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19148/HighFidelity-Beta-PR15495-ecc38a8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19148/HighFidelity-Beta-Interface-PR15495-ecc38a8.exe

**Comment by Atlante45** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19157/HighFidelity-Beta-PR15495-ecc38a8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19157/HighFidelity-Beta-Interface-PR15495-ecc38a8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19157/HighFidelity-Beta-PR15495-ecc38a8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19157/HighFidelity-Beta-Interface-PR15495-ecc38a8.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19157/HighFidelity-Beta-PR15495-ecc38a8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19157/HighFidelity-Quest-Beta-PR15495-ecc38a8.apk)

File: libraries/networking/src/udt/PacketHeaders.h (hunk 1 above)
Comment (ZappoMan) [reply libraries/networking/src/udt/PacketHeaders.h]: no required

**Pull request merged**